cmd/bpfman-operator/main.go

```diff
@@ -170,6 +170,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&bpfmanoperator.BpfApplicationReconciler{
```

Comment:
I think we're going to want to get rid of the per-program type reconcilers and just have this one.

Reply:
the loop will be running for each type but we will doc we can either use aggr type or the different obj if we need to support both I think ?

Reply:
I was thinking we would have one controller/Reconcile() function for bpfman-operator and one for bpfman-agent that could handle both new new bpfapplication crd and the old *Program CRDs.  Ideally, we could still use a lot of the code that exists, but new controllers would loop through the programs and handle them one at a time.  I'm going to have to think about it some more though.